test/jasmine/assets/touch_event.js

```diff
@@ -9,6 +9,10 @@ module.exports = function(type, x, y, opts) {
         target: el,
         clientX: x,
         clientY: y,
+        screenX: x,
```

Comment:
I was forced to complete the signature of the event or simulating the touch event on the slider will generate a very strange behavior (slider was always moved on the start position).

Reply:
Do we really need to add these? It looks the test passes without it.

Reply:
@archmoj not on my machine (MacOS). Just tested again removing these lines and I have the failure i fixed adding this

Reply:
BTW: I'm able to reproduce the issue also outside Jasmine/jsdom.

![issue](https://user-images.githubusercontent.com/284924/128391747-8cd86339-4fbc-48e4-993d-b3a7dbec2693.gif)

I'm 100% for removing this if you think it's just an issue on my Chrome… but seems really strange to me.

Reply:
Thanks for testing.
LGTM :medal_sports: 
Over to @alexcjohnson